python/lsst/ip/diffim/zogy.py

```diff
@@ -687,6 +723,20 @@ def computeScorrFourierSpace(self, xVarAst=0., yVarAst=0., **kwargs):
         - S_var : the corrected variance image (denominator of eq. 25 of ZOGY (2016))
         - Dpsf : the PSF of the diffim D, likely never to be used.
         """
+        # Some masked regions are NaN or infinite!, and FFTs no likey.
+        def fix_nans(im):
+            """Replace any NaNs or Infs with the mean of the image."""
+            isbad = np.isinf(im) | np.isnan(im)
```

Comment:
Consider `isbad = not np.isfinite(im)` as it handles a few more cases

Reply:
Good advice! Should that be ~np.isfinite(im)  (element-wise negation)?

Reply:
Yes. Pity it looks like bitwise negation. `np.logical_not(...)` is more verbose but unambiguous. You decide.